.changes/unreleased/tinymce-TINY-11405-2024-10-20.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: 'Improved dark skin for Tiny Comments plugin.'
```

Comment:
Not sure where we should stick this comment it's a fix for the comments plugin but it needs to be in core. 🤔

Reply:
Removed ✅